src/pyfreedb/providers/google/sheet/wrapper.py

```diff
@@ -173,17 +173,14 @@ def _parse_cell(self, cell: Dict[str, str], col: Dict[str, str]) -> Any:
         if typ == "boolean":
             return cell["v"]
         elif typ == "number":
-            if "f" in cell:
-                if "." in cell["f"]:
-                    return float(cell["f"])
-
-                return int(cell["f"])
-
-            # Computed data doesn't have raw value, number returned from aggregation will doesn't have `f`.
-            return int(cell["v"])
+            # Internally, google sheet represent number according to IEEE-754.
+            # If the actual value is outside the supported range it will be truncated.
+            return cell["v"]
```

Comment:
We can't do this in this layer, so wrapper will always return float regardless of the actual field's type. I think this is also better because now the wrapper doesnt have to know the typecasting logic.

When we load the data to the model, we will do the actual typecasting.